incremental-font-transfer/src/glyph_keyed.rs

```diff
@@ -100,7 +102,32 @@ pub(crate) fn apply_glyph_keyed_patches(
         }
     }
 
-    // TODO(garretrieger): mark the patch applied in the appropriate IFT table.
+    // Mark patches applied in IFT and IFTX as needed, copy the modified tables into the font builder.
+    let mut new_itf_data = font
+        .table_data(Tag::new(b"IFT "))
```

Comment:
I'd probably declare these as `const` somewhere? you use them a few times.

Reply:
Agreed, I'll do that in a separate PR.